docs/developers/inter-app-messaging.md

```diff
@@ -0,0 +1,56 @@
+# Inter-app messaging with SQS and SNS
```

Comment:
Do you really think we need to explain this? Could we not point to resources about the pubsub pattern & the AWS docs?

It's a good explanation, I suppose whether or not we need it depends on the audience.

Reply:
Whether or not we do, perhaps worth referencing some [nice explanation of pubsub](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern).

Reply:
I'm assuming an audience who isn't super familiar with the internals of AWS – this is (like a bunch of the other documentation) meant to be a bit of a reading list.

I do think the SQS-backed autoscaling is somewhat unusual and worth mentioning.